endpoints/api/user.py

```diff
@@ -480,6 +480,7 @@ def put(self):
 
     @show_if(features.USER_CREATION)
     @show_if(features.DIRECT_LOGIN)
+    @require_user_admin
```

Comment:
Why is this added?

Reply:
the `get_authenticated_user()` was returning a `NoneType` exception without it

Reply:
Which is expected if this is called by an anonymous user.

Reply:
Say when a new user is registering for the first time.

Reply:
ah, yes, I've updated it. PTAL